user-interface/src/main/java/life/qbic/datamanager/views/projects/create/AddProjectDialog.java

```diff
@@ -306,4 +298,80 @@ public CancelEvent(AddProjectDialog source, boolean fromClient) {
       super(source, fromClient);
     }
   }
+
+  /**
+   * <b>Project Creation Information</b>
+   *
+   * <p>Down to earth project info data container.</p>
+   *
+   */
+  public static final class ProjectCreationInformation implements Serializable {
```

Comment:
Why not a record?

Reply:
Valid Point, i thought it's easier to understand since the object is filled step by step but it doesn't matter in the end